auth/tests/unit/token_test.py

```diff
@@ -24,7 +24,7 @@ def test_importable():
         'auth_provider_x509_cert_url': 'https://www.googleapis.com/oauth2/v1/certs',
         'client_x509_cert_url': 'https://www.googleapis.com/robot/v1/metadata/x509/gcloud-aio%40random-project-123.iam.gserviceaccount.com'
     }
-    service_file = io.StringIO(json.dumps(service_data))
+    service_file = io.StringIO('{}'.format(json.dumps(service_data)))
```

Comment:
I can't even recall doing this, lol, thanks for noticing!

Reply:
Okay, I remember this now. So when we run `py3to2` in CI, `future-fstrings-show` redefines `str` literals to `unicode`. That works great in this case because `io.StringIO` does not like `str` inputs in `python2`. So this noop operation actually helps out with that.

I think a comment explain this will be worthwhile?